src/scripts.js

```diff
@@ -130,6 +134,7 @@ function displayRecipeDetailPage(event) {
     if (user.recipesToCook.length > 0 && user.recipesToCook.includes(foundRecipe)) {
         show(removeRecipeButton)
     } else {hide(removeRecipeButton)}
+    currentView.innerText = foundRecipe.name
```

Comment:
I don't think that we need the title of the recipe here, I would just do `currentView.innerText = ""`